### ISSUE TITLE
Release flatpak version of v1.19.x

### ISSUE THREAD

**Issue by igor2x** [pwsafe/pwsafe#1320]
On [Flathub there is still Password Safe v1.18.2](https://flathub.org/apps/org.pwsafe.pwsafe) despite v1.19 was released 3 weeks ago. Can you please build official flatpak and publish it on Flathub.

---

But before creating flatpak I suggest to merge [PR 1310](https://github.com/pwsafe/pwsafe/pull/1310) into current Linux source code. This PR fixes pretty nasty bug on Wayland of not coping password at all on e.g. Fedora like described in [issue 1307](https://github.com/pwsafe/pwsafe/issues/1307)  or even crashes Ubuntu like in [issue 
1308.](https://github.com/pwsafe/pwsafe/issues/1308) Which makes current v1.19 pretty much not really useful. User is required to see the password and type in some form manually.

---

Flatpak should be easy to build in manifest file [pwsafe flatpak manifest file](https://github.com/flathub/org.pwsafe.pwsafe/blob/master/org.pwsafe.pwsafe.yml) just two changes are required. In line 97 set new tag from Github commit and 98 set the actual Github commit number.

Then build flatpak according to official Password Safe [flatpak instructions.](https://github.com/pwsafe/pwsafe/blob/master/README.LINUX.FLATPAK.md).

**Comment by igor2x** [comment]
@rony, I have tested Password Safe v1.19.1 using flatpak on Ubuntu 24.04 and I think it is working fine.
Can you please build flatpak and push it to Flathub.org.

I tested:
1. Download [manifest file.](https://github.com/flathub/org.pwsafe.pwsafe/blob/master/org.pwsafe.pwsafe.yml)
2. Open file with text editor and find lines 97 and 98:
```
        tag: 1.18.2fp4                                  
        commit: 4062b9374538992883d7cde0da096f803ba2a835
```
3. Replace above lines with:
```
        tag: 1.19.1                                     
        commit: 0496dbf52fae9c152b6707b1b0ab0abb7ba13101
```
4. Then build flatpak using official [instructions.](https://github.com/pwsafe/pwsafe/blob/master/README.LINUX.FLATPAK.md)

**Comment by igor2x** [comment]
I have performed additional testing and I see only English is available from Manage | Select Language. To fix this issue I have removed lines 31 and 34 from manifest file just like in [PR 8](https://github.com/flathub/org.pwsafe.pwsafe/pull/8/files) on Github  flathub's org.pwsafe.pwsafe that is waiting to be merged. Building flatpak and problem is solved.

After above "fix" languages looks fine now:
![image](https://github.com/pwsafe/pwsafe/assets/10895030/2621a433-232a-415b-a00b-0bc7b57f0a08)

**Comment by igor2x** [comment]
@ronys, I see to build flatpak you have updated manifest file with  v1.19.1 commit. This is great. Thanks.

1. Can you also please merge [PR 8](https://github.com/flathub/org.pwsafe.pwsafe/pull/8) into master, so translations will work.
2. Publish flatpak to Flathub.org.

**Comment by igor2x** [comment]
I see from GUI that new flatpak version was pushed to Flathub.org.
![image](https://github.com/pwsafe/pwsafe/assets/10895030/5265773b-2aa7-467d-bac4-9b9fb4b2a5de)

But looking to Flathub, there is still v1.18.2 version displayed.
![image](https://github.com/pwsafe/pwsafe/assets/10895030/ca169799-e5e6-4e09-8b14-28dd2e4411f5)

If I execute: `flatpak info org.pwsafe.pwsafe` then I get mix of v1.18.2 and commit 1.19.1.

![image](https://github.com/pwsafe/pwsafe/assets/10895030/9d041bf4-84c8-472c-b002-e1a1d70824cc)

Somehow "Version" has to be changed. Maybe this info is not in manifest file, but some upstream repository file like this one: [install/metainfo/org.pwsafe.pwsafe.metainfo.xml](https://github.com/pwsafe/pwsafe/blob/121cd42d3765089b0154ded8c308f3b8361b9330/install/metainfo/org.pwsafe.pwsafe.metainfo.xml#L31)
![image](https://github.com/pwsafe/pwsafe/assets/10895030/57235d55-3267-4afb-9789-9cbbf3b26145)

Beside version number, please also change release date. Both info is displayed on Flathub.org.

**Comment by ronys** [comment]
Fixed in commit bfa775242
